pycbc/strain/strain.py

```diff
@@ -796,6 +796,11 @@ def insert_strain_option_group_multi_ifo(parser, gps_times=True):
                                     help='Ignore the given length of whitened '
                                          'strain at the ends of a segment, to '
                                          'avoid filters ringing.')
+    data_reading_group_multi.add_argument('--gating-method', type=str,
+                                    default='taper',
```

Comment:
This does need the `nargs="+"` and `action=MultiDetOptionAction,` kwargs to work properly with the multi-ifo options stuff.

Reply:
Good to know, I've just added them within my branch. I'm not sure if that warrants a new pull request or if it can be done here.

Reply:
Thanks @jakeb245 I think now that this is merged this needs to be a new pull request (in theory one could mess around with the git history on the main gwastro/pycbc branch, but that's a bad idea in practice!)

Just to add context here. The multi-ifo options allow for the ability to provide different values for different ifos. So for *all* options, you [should be able to] do either:
`--gating-method taper` (set for all ifos) or `--gating-method H1:taper L1:something_else`. Then at the code end you normally need to do:
`opt.gating_method[IFO_NAME]`
to get what you want.

Now the strain `from_cli` interface is designed to work in either case, so it looks for either `opt.gating_method[IFO_NAME]` *or* `opt.gating method` (and the same for all other options). However, this is a special case. The inference code (I think) does not do this, and would fail when it tries to do `opt.gating_method[IFO_NAME]`.

Reply:
Sorry for not catching this, and yes, I agree this needs a new pull request.